core/package-lock.json

```diff
@@ -48,6 +48,7 @@
       }
     },
     "custom-rules": {
+      "name": "eslint-plugin-custom-rules",
```

Comment:
Running `npm i @stencil/core` sets this. I think it's just a remnant of another change that didn't propagate to the lockfile. LMK if you wanna split this change out